controllers/userC.js

```diff
@@ -22,5 +22,27 @@ const createUser = asyncWrapper(async(req, res, next) => {
 
     res.status(201).json({ msg: 'User created successfully', user: newUser });
 });
+ const userLogin = asyncWrapper(async (req, res, next) => {
+    const userLoginRequest = { email: req.body.email, password: req.body.password };
+    const user = await fetchUser({ email: req.body.email });
+    if (!user) {
+        return next(createCustomError("User not found", 404));
+    }
+
+    const correctPassword = await bcrypt.compare(userLoginRequest.password, user.password );
+    if (correctPassword === false) {
+        return next(createCustomError('Invalid email or password', 404));
```

Comment:
Find the correct status code for invalid password and use it.
It is not 404.

Reply:
Okay, 401 is the correct code